### ISSUE TITLE
vue/html-indent no work

### ISSUE THREAD

**Issue by ShumRain** [vuejs/eslint-plugin-vue#822]
**Tell us about your environment**
* **macOS:** 10.14.2
* **VScode:** 1.31.0
* **ESLint version:** 4.19.1
* **eslint-plugin-vue version:** 4.7.1
* **Node version:**	

**Please show your full configuration:**

```javascript
module.exports = {
    root: true,
    parserOptions: {
        parser: 'babel-eslint',
        sourceType: 'module'
    },
    env: {
        browser: true,
        node: true,
        es6: true,
    },
    extends: ['plugin:vue/recommended', 'eslint:recommended'],

    rules: {
        'vue/max-attributes-per-line': [2, {
            'singleline': 3,
            'multiline': {
                'max': 1,
                'allowFirstLine': false
            }
        }],
        'vue/name-property-casing': [2, 'PascalCase'],
        'vue/html-indent': [2, 4, {
            'attribute': 1, 
            'baseIndent': 0, // no work
            'closeBracket': 0,
            'alignAttributesVertically': true
        }],
        'vue/script-indent': [2, 4, {
            'baseIndent': 0
        }],
    }
}
```

**What did you do?**	

```vue
<template>
    <div id="app">
        <router-view />
    </div>
</template>
```

**What did you expect to happen?**
I hope to see the error tip
like this 
![image](https://user-images.githubusercontent.com/24872072/52944380-3ea57b00-33aa-11e9-8bb2-ca6cb3e8ec4b.png)

**What actually happened?**

but I get nothing 
Other options work fine

**Comment by mysticatea** [comment]
Thank you for this issue.

But I'm confused. Which do you talking about `vue/html-indent` rule or `vue/script-indent` rule?
Your answers of "What did you do?" and "What did you expect to happen?" are mismatched.

**Comment by mysticatea** [comment]
Ah... I understand you now.

The `baseIndent` option of `vue/html-indent` has been added in `eslint-plugin-vue@5.0.0`.
Please use the latest version.
